Flow.Launcher.Core/Plugin/PluginsLoader.cs

```diff
@@ -50,27 +53,43 @@ public static IEnumerable<PluginPair> DotNetPlugins(List<PluginMetadata> source)
                     }
                     catch (Exception e)
                     {
+                        erroredPlugins.Add(metadata.Name);
+
                         Log.Exception($"|PluginsLoader.DotNetPlugins|Couldn't load assembly for {metadata.Name}", e);
                         return;
                     }
-                    var types = assembly.GetTypes();
+
                     Type type;
                     try
                     {
+                        var types = assembly.GetTypes();
+                        
                         type = types.First(o => o.IsClass && !o.IsAbstract && o.GetInterfaces().Contains(typeof(IPlugin)));
                     }
                     catch (InvalidOperationException e)
                     {
+                        erroredPlugins.Add(metadata.Name);
+
                         Log.Exception($"|PluginsLoader.DotNetPlugins|Can't find class implement IPlugin for <{metadata.Name}>", e);
                         return;
                     }
+                    catch (ReflectionTypeLoadException e)
+                    {
+                        erroredPlugins.Add(metadata.Name);
+
+                        Log.Exception($"|PluginsLoader.DotNetPlugins|The GetTypes method was unable to load assembly types for <{metadata.Name}>", e);
```

Comment:
Where this error will be seen? If the user will see it, perhaps we could try making it to look as a solution, like pointing the user where to fix it?

Reply:
All those errors were logged previously and not visible to the user. And this uncaught exception would just fail the app completely.

For this PR, it simply puts out a message to let user know the failed plugins, and continue to start up, I think it's the simplest approach.

Failure of loading could be due to a number of reasons , but none a non dev user would not know how to fix it. I think reporting it might be the best approach?

Reply:
We could add something like "This is an error caused by the plugin X" so the user knows that it is something wrong with a plugin, rather than with their installation

Reply:
This is ok?

![image](https://user-images.githubusercontent.com/26427004/82604860-46c91780-9bf8-11ea-87cd-83cb8386ce61.png)

Reply:
I like it! 
Maybe we could also hint the user about taking a look into logs? Like:
<<your current message>> Please refer to the logs for more information 
Potentially you could also include some styling for the message box like: ```MessageBoxIcon.Warning```

Reply:
Updated:

![image](https://user-images.githubusercontent.com/26427004/82859498-39928c80-9f5a-11ea-8744-95c7a2bbb1d1.png)

Reply:
@gdziedzic please re-review